variables.tf

```diff
@@ -50,3 +50,28 @@ variable "public_ssh_key" {
   default     = ""
 }
 
+variable "os_disk_size_gb" {
+  description = "Disk size of nodes in GBs"
+  type        = number
+  default     = 50
+}
+
+variable "network_profile" {
+  description = "Variables defining the AKS network profile config"
+  type = object({
+    network_plugin     = string
+    network_policy     = string
+    dns_service_ip     = string
+    docker_bridge_cidr = string
+    pod_cidr           = string
+    service_cidr       = string
```

Comment:
Could we create these resources in this module instead of importing them as variables?